IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Documentation.md

```diff
@@ -0,0 +1,19 @@
+The _IfcSharedInfrastructureElements_ schema defines the shared or common elements (subtypes of _IfcElement_) that are applicable to the infrastructure sector.
+
+the scope of this includes concepts to cover the definition of:
+
+* Geology & terrain
```

Comment:
Not sure about geology and earthworks.

Reply:
well the earthworks entities are in this package, i changed geology to geotechnical

Reply:
Yes, of course. Let us not start any discussion on this now. Treat my comment as an observation for traceability.